### PR TITLE
Refactors some of the surgery code to be more cleaner

### PR DESCRIPTION
**Pull request by MaxIsJoe**
There are no gameplay changes in this PR.

### CODE REVIEW

File: UnityProject/Assets/Scripts/HealthV2/Living/Surgery/Dissectible.cs [hunk 1]
```diff
@@ -164,98 +154,106 @@ public void SetBodyPartID(uint oldState, uint newState)
...
 
-				if (LivingHealthMasterBase.GetComponent<PlayerSprites>().RaceBodyparts.Base.RootImplantProcedure ==
-				    SurgeryProcedureBase)
-				{
-					this.currentlyOn = LivingHealthMasterBase.gameObject;
-					this.ThisPresentProcedure.SetupProcedure(this, null, SurgeryProcedureBase);
-				}
-			}
+		// Setup the surgery procedure when the body part is closed
+		private void SetupSurgeryProcedureWithoutSetup(SurgeryProcedureBase surgeryProcedureBase, BodyPart bodyPart)
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/HealthV2/Living/Surgery/Dissectible.cs:224]: setup something without setup?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/HealthV2/Living/Surgery/Dissectible.cs:224]: The code was nonsensical from the start, I don't expect anyone or even an LM to be able to fully undertand what is going on in ServerCheck, but it's better than the previous state it was in

Comment (corp-0) [reply UnityProject/Assets/Scripts/HealthV2/Living/Surgery/Dissectible.cs:224]: Good thing the author is alive and you can ask him what was his intention

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/HealthV2/Living/Surgery/Dissectible.cs:224]: Bod doesn't know what they do
he said it's fiddly and he hasn't given me names

**Pull request merged**